crates/experimental-api/src/gen.rs

```diff
@@ -1659,6 +1676,12 @@ pub mod gen {
                 &self.0
             }
         }
+
+        impl From<StorageKey> for String {
+            fn from(value: StorageKey) -> Self {
+                value.0
```

Comment:
There is already `impl AsRef` doing exactly this

Reply:
Removed